app/javascript/components/schedule-form/schedule-form-fields.js

```diff
@@ -90,7 +90,7 @@ export const systemField = (data) => ({
   component: componentTypes.SELECT,
   id: 'system',
   name: 'instance_name',
-  label: __('System/Proess'),
+  label: __('System/Process'),
```

Comment:
I think this is weird to be translated, because it's referencing an actual automate instance which is not translated.

(Not necessarily for this PR).

Alternateively, maybe this should say something like "Automate Instance" (or whatever text we use elsewhere when someone chooses an automate instance).